sys-block/targetcli-fb/targetcli-fb-2.1.48-r3.ebuild

```diff
@@ -17,7 +17,7 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+BDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```

Comment:
DEPEND

Reply:
I followed [*The ultimate guide to EAPI 7*](https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#build-time-dependencies-split-into-depend-and-bdepend):

>1. Dependencies that need to be run during the build, and therefore must run on the system used to build packages (CBUILD). Those include toolchain, build system tooling (autotools, CMake), various language interpreters (Perl, Python), preprocessors (SWIG) and other tools (e.g. pkg-config). Those are placed in BDEPEND now.
>2. Dependencies that need to be compiled for the real system, and present for the toolchain to work. Those mostly include libraries since the link editor needs to link to them. Those remain as DEPEND.

While it doesn't list it explictly, wouldn't setuptools fall under "build system tooling", and thus go into BDEPEND?